lib/membrane_mp4/payloader/aac.ex

```diff
@@ -38,6 +38,8 @@ defmodule Membrane.MP4.Payloader.AAC do
 
   @impl true
   def handle_process(:input, buffer, _ctx, state) do
+    # TODO: remove once bumping to a version of AAC plugin that supports buffers with dts/pts
+    buffer = Map.put(buffer, :dts, buffer.metadata.timestamp)
```

Comment:
I believe we should use pts for audio instead of dts as there is no concept of decoding and presentation order for audio AFAIK (maybe there are some cases - actually why not :D). Can you synchronize this change with @daniel-jodlos, I believe he is working on the support for dts and pts in the AAC plugin?

Reply:
When building sample tables, MP4 specification requires calculating deltas between consequent DTS-es (you can look up e.g. `stts` definiton), and modules were build with that in mind. @daniel-jodlos changes assume putting timestamps only in `:pts`, so we can go either with:
- setting dts=pts in buffers flowing through `Payloader.AAC`, so that modules creating sample tables can handle samples no matter if they are H264 or AAC (preferred)
- going back to notion of "timestamps" as an umbrella term for DTS and PTS and use `Buffer.get_dts_or_pts/1`

IMHO the second option, even if technically more correct, would introduce too much chaos in the plugin

Reply:
Ok, both options look good, if you think the first one leads to a cleaner solution, let's go with that.